ui/src/organizations/actions/orgs.ts

```diff
@@ -80,6 +81,9 @@ export interface SetOrg {
 }
 
 export const setOrg = (org: Organization): SetOrg => {
+  HoneyBadger.setContext({
```

Comment:
when a user changes orgs / logs out and logs in as someone else, does this just overwrite in honey badger?

Reply:
@drdelambre great question, love where your head is at. The docs don't specifically mention this, but from what I've seen, i'm fairly certain that calling `setContext` will overwrite an existing key, but merge new keys.

>Use `Honeybadger.setContext` to add context to error reports. You can call `setContext` as many times as you like, and it will be reset on each page load. New context data is merged with existing data:

https://docs.honeybadger.io/lib/javascript/guides/customizing-error-reports.html#custom-metadata-context

I don't think we want to clear it though, because it will clear out all the keys:

>If you've used `Honeybadger.setContext` to store context data, you can clear it with `Honeybadger.resetContext`